### ISSUE TITLE
Keep getting ERR_EMPTY_RESPONSE errors

### ISSUE THREAD

**Issue by mbootsman** [vagrantpress/vagrantpress#68]
When requesting urls, about 33% of the times I get a ERR_EMPTY_RESPONSE error code in the browser. After refreshing for about 2-3 times, the page is loaded. Googled it but found no solution. 

Done a forced provision, still get this. Any help is greatly appreciated.

Error Message:
No data received
Error code: ERR_EMPTY_RESPONSE

**Comment by beipawel** [comment]
Any progress with this one? I installed the newest versions of Vagrant and VirtualBox, still getting `err_empty_response` errors. 

![image](https://cloud.githubusercontent.com/assets/283810/6091539/21c484cc-aeb1-11e4-930e-a996944bdee1.png)

**Comment by timlinde** [comment]
All you need todo is to go to wp-login.php and select a theme.

**Comment by beipawel** [comment]
How do I do that? 
I'm just starting out with WP and I can't find the line where I can select a theme in wp-login.php.

**Comment by vladiiancu** [comment]
I think he wanted to say that you should login in WordPress Admin panel and go to appearance and select and apply a theme.

**Comment by beipawel** [comment]
The Problem I had was, that I changed the Domainname in the `Vagrantfile`. The `wordpress-db.sql` has the Domain `vagrantpress.dev` hardcoded though. I changed that in the `wordpress-db.sql` file and did a `vagrant up --provision` which didn't work (I thought it will drop the table `wp_options`, but it seems it didn't). I then used phpmyadmin (which worked fine) and changed the domain name in the table `wp_options` manually. Now it works.

**Comment by chad-thompson** [comment]
^^ thanks for finding that fix.  I'm hoping to get some of the 'hardcoded' items removed here in the near future.

**Comment by beipawel** [comment]
Don't know how Vagrantfile works exactly, but maybe you could set a variable with the domain name and then after setting up the wordpress-db.sql (with some dummy domain name) you could prepare a dynamic sql update statement with the domain name from the variable. Like

    /usr/bin/mysql -u wordpress -pwordpress wordpres -e "update wp_options set option_value='http://<domainname_var>' where option_name in ('siteurl', 'home');"

That way you would not need to change the wordpress-db.sql file. Just a thought.

**Comment by battlemidget** [comment]
Probably need some sort of "hook" that when the $domainname changes to alter the database and update its domainnames.

**Comment by Johnstonian** [comment]
good idea @beipawel and @battlemidget, shouldn't be too hard to implement. agree with @chad-thompson, some work could be done to make things less 'hardcoded'.

**Comment by Johnstonian** [comment]
in regards to the ERR_EMPTY_REPSONSE problem. After some googling, it seems like this is an issue with chrome's temporary internet files. Suggest clearing cache and history. I had this issue earlier today. It also seems like it occurs if I put my laptop to sleep when a vagrant box is running. I boot it back up and I get the ERR_EMPTY_RESPONSE messages in Chrome. That can be fixed by suspending or destroying the vagrant box. Restart the host machine. Then run vagrant up.

**Comment by rita2clark** [comment]
Some days ago i faced this error. you can clear your cache and change Lan settings and get rid of this error.
 
#customize and Control
#Settings
#Show Advanced settings
#Privacy section
#Clear browsing data.
 
This error occurs for many reason thats why you check all the solution of this  '<a href='http://www.toppctech.com/fixing-err-empty-response-error/'>ERR_EMPTY_REPSONSE </a>' error from here.

**Comment by Jason383** [comment]
How To Fix ERR_EMPTY_RESPONSE: https://www.wiknix.com/solved-err_empty_response-no-data-received/
